content/Education and Kits/Science Journal App/Upload-the-Science-Journal-firmware.md

```diff
@@ -59,17 +65,23 @@ Find and install the libraries by using the Library Manager (`Tools > Manage Lib
 
 1. Find and open the firmware upload sketch:
```

Comment:
I think this part can be simplfied into a list of items like these:

> <Hardware>: <Example path>

We can explain once that in IDE 1, the libraries will be located under INCOMPATIBLE.

Reply:
Just an idea, perhaps a table like this can be useful...

| Board                                            | Library                | Example                                |
|--------------------------------------------------|------------------------|----------------------------------------|
| Arduino Nano 33 BLE Sense (by itself)            | Arduino_ScienceJournal | /ScienceKit/Nano33BLESenseFirmware     |
| Arduino Nano 33 BLE Sense Rev2 (by itself)       | Arduino_ScienceJournal | /ScienceKit/Nano33BLESenseRev2Firmware |
| Arduino MKR WiFi 1010 (with Science Carrier)          | Arduino_ScienceJournal |         /ScienceKit/PhysicsLabFirmware |
| Arduino MKR WiFi 1010 + (with Science Carrier Rev2)     | Arduino_ScienceJournal |       /ScienceKitR2/PhysicsLabFirmware |
| Arduino Nano RP2040 Connect (by itself)          |                      ... |                                      ... |
| Arduino Nano RP2040 Connect + (with Science Carrier R3) |                      ... |                                      ... |

Reply:
For the last ones:

Arduino Nano RP2040 Connect (by itself) |Arduino_ScienceJournal | /ScienceKit/RP2040ConnectFirmware

Arduino Nano RP2040 Connect (with Science Carrier R3) | Arduino_ScienceCarrier | /ScienceJournal